product_variant_multi_link/models/product_product.py

```diff
@@ -14,6 +15,47 @@ class ProductProduct(models.Model):
         compute="_compute_product_link_ids",
     )
 
+    product_product_link_count = fields.Integer(
+        string="Product Links Count", compute="_compute_product_product_link_count"
+    )
+
+    @api.depends("product_variant_link_ids")
+    def _compute_product_product_link_count(self):
+        link_model = self.env["product.template.link"]
+        # Set product_template_link_qty to 0 if user has no access on the model
+        try:
+            link_model.check_access_rights("read")
```

Comment:
or you can add a group right on xml view so for the rest of the users button will be invisible

Reply:
then if you use the same field in another view you won't have the same result